Documentation/cad-models.rst

```diff
@@ -4,10 +4,20 @@ CAD models
 The following help pages describe how to build a structural model with
```

Comment:
Maybe add some remark about when to use Simmodeler and when to use Gocad here.

Reply:
I think gocad can still be useful for some complex surface processing, such as surface smoothing with constrains. But this may be too technical, so I leave that unchanged.

Reply:
Ok I added (e.g. surface smoothing with constraints) as an example below.